### DISCUSSION TITLE
sci_oxy3835_wphase_oxygen and sci_oxy4_oxygen parameters ?

### DISCUSSION THREAD

**Discussion by vturpin** [OceanGlidersCommunity/Oxygen_SOP#190] [Q&A] [answered]
Hi,
I have a technical question regarding oxygen sensor on slocum :
Do you know what exactly are the variables sci_oxy3835_wphase_oxygen and sci_oxy4_oxygen (output from the glider) corresponds to?

We thought it was: Uncompensated (pressure and salinity) oxygen concentration reported by the oxygen sensor But when we try to compute DOXY (Oxygen), it seems to be the case for sci_oxy4_oxygen but not for sci_oxy3835_wphase_oxygen.

For sci_oxy3835_wphase_oxygen , we have the impression that it's an Oxygen with salinity compensated but we are not sure. So, if you have any information about these variables, it would be really useful.
Thanks.

**Comment by soerenthomsen** [comment]
@gkrahmann @tomhull any idea?

**Comment by gkrahmann** [comment, marked as answer]
sci_oxy3835_wphase_oxygen
is a direct output from the optode. Since no information is fed from the glider into the optode, this is a non-compensated value.
There is no difference to  sci_oxy3835_oxygen apart from a different setting internal to the optode (wphase spits out more information).

When trying to reproduce the calculation of the optode (convert phase(s)&temp to oxygen) you however need some internal settings of the optode, which the glider and the glider's data files are unaware of !!! That is e.g. the internal salinity setting of the optode. I do not recall what the recommendation was. Either 0 or 35 PSU. Try using the either of these values. Or maybe somebody tried to be very smart and set this to something different. Of course you also must use the foil coefficients that were present in the optode during the measurements.
